bin/adhoc-scripts/createPileupObjects.py

```diff
@@ -11,6 +11,7 @@
 where the pileup document minimal schema is:
 {
     'pileupName': "",
+    "customName": "",
```

Comment:
this new attribute isn't mandatory (from the client perspective), so perhaps we could remove it from here?

Reply:
ok, you're right it should be optional